docs/monthly-meeting/2022-11.md

```diff
@@ -0,0 +1,65 @@
+# Documentation Community Team Meeting (November 2022)
+
+- **Date:** 2022-11-07
+- **Time:** [19:30 UTC](https://arewemeetingyet.com/UTC/2022-11-07/19:30/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- [**Discourse thread**](https://discuss.python.org/t/20057) (for November)
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [hackMD document]  (https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+- Mariatta
+- Jim DeLaHunt
+- Ezio
+- Erlend
+- Hugo
+- Ned
+- CAM
+- Petr (second half)
+
+
+## Quick updates - Introductions
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+- Mariatta: Python BR keynote 
+- Jim DeLaHunt: pass
+- Ezio: devguide, [CSV with status of Python branches](https://github.com/python/devguide/pull/884), [OpenGraph metadata for devguide et al.](https://github.com/python/devguide/pull/953), catching up after a busy month
+- Erlend: getting help from Cam on two huge PRs on SQLite docs
+- Hugo: OpenGraph extension on DevGuide, created PR on Sphinx extension to improve SEO, now does `meta descrption` in addition to `meta og:...`, plus bug fix in `dirhtml` used to generate the pages.
+- Cam: attended core dev sprint, helped a lot of people get things done. Will work on whats new. New Sphinx roles for linking Python source code & GitHub. automate checking if things added in new version is documented, docstrings updated, deprecation tracking.
+
+## Agenda items
+
+* (Alex) ["object doesn't have an `__enter__`/`__exit__` yet docs say it does"](https://github.com/python/cpython/issues/97699) might be interesting to discuss
+
+    - concern with renaming/changing text will break links (specifically, fragment parts of URL e.g. `#object.__enter__`).
+    - it's better to add a paragraph above explaining it. even though readers may not read the paragraph, at least we can still point people to it, instead of breaking links.
+
+
+- Infrastructure & detecting/avoiding broken links
+    - how to get visibility of docs traffic and broken url hits? 
+        - First step: ask [#PSF category on Discourse](https://discuss.python.org/c/python-software-foundation/9). 
+        - Done, see https://discuss.python.org/t/data-on-requests-for-missing-urls-in-docs-python-org-etc/20841 —JDLH
```

Comment:
```suggestion
        - Done, see [DIscourse thread](https://discuss.python.org/t/data-on-requests-for-missing-urls-in-docs-python-org-etc/20841) —JDLH
```

Make this an inline rather than raw link